etrago/appl.py

```diff
@@ -87,9 +87,26 @@
         'n_init': 10, # affects clustering algorithm, only change when neccesary
         'max_iter': 100, # affects clustering algorithm, only change when neccesary
         'tol': 1e-6,}, # affects clustering algorithm, only change when neccesary
+    'sector_coupled_clustering': {
+        'active': True, # choose if clustering is activated
+        'carrier_data': { # select carriers affected by sector coupling
+            'H2_ind_load': {
+                'base': ['H2_grid'],
+                'skip': None,
+            },
+            'central_heat': {
+                'base': ['CH4'],
+                'skip': 'central_heat_store'
+            },
+            'rural_heat': {
+                'base': ['CH4'],
+                'skip': 'rural_heat_store'
+            },
+        },
+    },
```

Comment:
What do you think about setting these parameters as default (inside networkclustering.py)?

Reply:
Why inside networkclustering and not appl.py?